### PR TITLE
fix(README): grammar

### PR DESCRIPTION
**Pull request by gargarchit**
Signed-off-by: Archit garg4.ag@gmail.com

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please Merge this

### CODE REVIEW

**Comment by gargarchit** [comment]
@irmerk Thanks for the review, I have update changes and added DCO

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Top level repository (cicero-ui), with sub packages. The entire package is publi
   <img src="assets/APLogo.png" alt="Accord Project Logo" width="400" />
 </a>
 
-Accord Project is an open source, non-profit, initiative working to transform contract management and contract automation by digitizing contracts. Accord Project operates under the umbrella of the [Linux Foundation][linuxfound]. The technical charter for the Accord Project can be found [here][charter].
+Accord Project is an open-source, non-profit, initiative working to transform contract management and contract automation by digitizing contracts. Accord Project operates under the umbrella of the [Linux Foundation][linuxfound]. The technical charter for the Accord Project can be found [here][charter].
```
Comment (irmerk) [inline README.md]: This can be reverted.

File: README.md [hunk 2]
```diff
@@ -333,7 +333,7 @@ Accord Project is an open source, non-profit, initiative working to transform co
 
 ## Contributing
 
-The Accord Project technology is being developed as open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
+The Accord Project technology is being developed as an open-source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
```
Comment (irmerk) [inline README.md]: This can be reverted.

**Comment by irmerk** [comment]
A couple changes, also make sure to rebase and push back up, then this can be merged.

File: README.md (hunk 1 above)
Comment (gargarchit) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (gargarchit) [reply README.md]: Done

**Comment by gargarchit** [comment]
@irmerk Done and Thanks for this

**Comment by gargarchit** [comment]
@irmerk I have Updated all changes

**Comment by gargarchit** [comment]
Thank You @irmerk

**Comment by irmerk** [comment]
Email jolene@clause.io so I can track your shipping info for Hacktoberfest.

**Comment by gargarchit** [comment]
Thank You @irmerk I will email now

**Pull request merged**